test_scripts/API/SameMenuName/002_CreateInteractionChoiceSet.lua

```diff
@@ -0,0 +1,51 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: SDL 0180 Broaden Choice Uniqueness
+-- 
+-- Description:
+--   Mobile shall be able to send a CreateInteractionChoiceSet RPC where two choices within the
+--   choice set have identical menuName values.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/Smoke/commonSmoke')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.syncMsgVersion.majorVersion = 5
+config.application1.registerAppInterfaceParams.syncMsgVersion.minorVersion = 0
```

Comment:
Is this needed?

Reply:
Nope, removed in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2501/commits/01e13168961d10a402f672c27fc818280773889a